Assets/HoloToolkit/ARCA/Plugins/WSA/x86/opencv_calib3d310.dll.meta

```diff
@@ -0,0 +1,34 @@
+fileFormatVersion: 2
```

Comment:
Is this a part of OpenCV? If so, we do not check external dependencies in to MRTK. You will need to document and call out this requirement and have users of ARCA provide them.

If this needs removal, your scripts will need to essentially noop if it cannot be found.